lapis/src/test/kotlin/org/genspectrum/lapis/silo/SiloClientTest.kt

```diff
@@ -169,9 +170,9 @@ class SiloClientTest(
         assertThat(
             result,
             containsInAnyOrder(
-                SequenceData(sequenceKey = "key1", sequences = mapOf("someSequenceName" to "ABCD")),
-                SequenceData(sequenceKey = "key2", sequences = mapOf("someSequenceName" to "DEFG")),
-                SequenceData(sequenceKey = "key3", sequences = mapOf("someSequenceName" to null)),
+                SequenceData(mapOf("primaryKey" to TextNode("key1"), "someSequenceName" to TextNode("ABCD"))),
```

Comment:
hmmm... can SILO ever return multiple segment sequences here? Maybe we should add a test for that

Reply:
Ah I see you did this in `lapis/src/test/kotlin/org/genspectrum/lapis/controller/MockData.kt` so feel free to ignore me :-)